youtube_dl/extractor/vice.py

```
@@ -21,10 +21,21 @@
 
 class ViceBaseIE(AdobePassIE):
     def _extract_preplay_video(self, url, locale, webpage):
-        watch_hub_data = extract_attributes(self._search_regex(
-            r'(?s)(<watch-hub\s*.+?</watch-hub>)', webpage, 'watch hub'))
-        video_id = watch_hub_data['vms-id']
-        title = watch_hub_data['video-title']
+        try:
+            watch_hub_data = extract_attributes(self._search_regex(
+                r'(?s)(<watch-hub\s*.+?</watch-hub>)', webpage, 'watch hub'))
+            video_id = watch_hub_data['vms-id']
+            title = watch_hub_data['video-title']
+        except:
```

Never use bare except.